src/bin/data/questions.js

```diff
@@ -0,0 +1,47 @@
+import validUrl from "valid-url"
+
+export const defaultAnswers = {
+  name: "Statinamic",
+  homepage: "http://moox.io/statinamic/",
+  CNAME: false,
+  nojekyll: true,
```

Comment:
nojekyll should (I think) not be asked and added by default. Enabled by default (already enabled if not provided, per "configurator") and just documented.